packages/dx-react-chart-bootstrap4/src/templates/tooltip/sheet.test.jsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Sheet } from './sheet';
+
+describe('Content', () => {
+  const defaultProps = {
+    className: 'custom_class_name',
+    placement: 'placement',
+  };
```

Comment:
These properties are not required. I think we should remove `defaultProps` variable.

The next tests have the same issue.